src/System.Numerics.Vectors/src/System/Numerics/Vector4_Intrinsics.cs

```diff
@@ -160,6 +160,24 @@ public static float Dot(Vector4 vector1, Vector4 vector2)
                    vector1.W * vector2.W;
         }
 
+        /// <summary>
+        /// Computes the cross product of two vectors. For homogeneous
+        /// coordinates, the product of the weights is the new weight
+        /// for the resulting product.
+        /// </summary>
+        /// <param name="vector1">The first vector.</param>
+        /// <param name="vector2">The second vector.</param>
+        /// <returns>The cross product.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static Vector4 Cross(Vector4 vector1, Vector4 vector2)
+        {
+            return new Vector4(
+                vector1.Y * vector2.Z - vector1.Z * vector2.Y,
+                vector1.Z * vector2.X - vector1.X * vector2.Z,
+                vector1.X * vector2.Y - vector1.Y * vector2.X,
+                vector1.W * vector2.W);
+        }
```

Comment:
In addition to tests, these would also need to be added to the relevant .cs file under \ref\.  Otherwise, they won't be exposed.